setup.py

```diff
@@ -10,7 +10,7 @@
 README = open(os.path.join(here, 'README.md')).read()
 CHANGES = open(os.path.join(here, 'CHANGES.md')).read()
 VERSION_BYTES = subprocess.check_output("git describe --tags --match '[0-9]*'", shell=True).strip()
```

Comment:
`VERSION_BYTES = subprocess.check_output("git describe --tags --match '[0-9]*'", shell=True, universal_newlines=True).strip()` also returns the result as text

Reply:
Good to know, thanks.